pronto_mia/lib/ui/views/deployment_plan/edit/deployment_plan_edit_view.dart

```diff
@@ -51,88 +57,124 @@ class DeploymentPlanEditView extends StatelessWidget
+            type: DateTimePickerType.dateTime,
+            controller: availableFromController,
+            firstDate: deploymentPlan != null
+                ? deploymentPlan.availableFrom
+                : DateTime.now(),
+            lastDate: DateTime.now().add(const Duration(days: 365)),
+            dateMask: 'dd.MM.yyyy HH:mm',
+            dateLabelText: 'Gültig ab*',
+          ),
+          DateTimePicker(
+            type: DateTimePickerType.dateTime,
+            controller: availableUntilController,
+            firstDate: deploymentPlan != null
+                ? deploymentPlan.availableUntil
+                : DateTime.now(),
+            lastDate: DateTime.now().add(const Duration(days: 365)),
+            dateMask: 'dd.MM.yyyy HH:mm',
+            dateLabelText: 'Gültig bis*',
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            crossAxisAlignment: CrossAxisAlignment.baseline,
+            textBaseline: TextBaseline.ideographic,
+            children: [
+              TextButton(
+                onPressed: () => handlePdfUpload(model),
+                child: const Text('Datei hochladen'),
               ),
-            ),
-            DateTimePicker(
-              type: DateTimePickerType.dateTime,
-              controller: availableFromController,
-              firstDate: deploymentPlan != null
-                  ? deploymentPlan.availableFrom
-                  : DateTime.now(),
-              lastDate: DateTime.now().add(const Duration(days: 365)),
-              dateMask: 'dd.MM.yyyy HH:mm',
-              dateLabelText: 'Gültig ab*',
-            ),
-            DateTimePicker(
-              type: DateTimePickerType.dateTime,
-              controller: availableUntilController,
-              firstDate: deploymentPlan != null
-                  ? deploymentPlan.availableUntil
-                  : DateTime.now(),
-              lastDate: DateTime.now().add(const Duration(days: 365)),
-              dateMask: 'dd.MM.yyyy HH:mm',
-              dateLabelText: 'Gültig bis*',
-            ),
-            Row(
-              mainAxisAlignment: MainAxisAlignment.center,
-              crossAxisAlignment: CrossAxisAlignment.baseline,
-              textBaseline: TextBaseline.ideographic,
-              children: [
-                TextButton(
-                  onPressed: () => handlePdfUpload(model),
-                  child: const Text('Datei hochladen'),
-                ),
-                const SizedBox(width: 8.0),
-                Expanded(
-                  child: TextField(
-                    controller: pdfPathController,
-                    readOnly: true,
-                    onTap: model.openPdf,
-                    style: const TextStyle(
-                      color: Colors.blue,
-                      decoration: TextDecoration.underline,
-                    ),
+              const SizedBox(width: 8.0),
+              Expanded(

```

Comment:
Can you remodel this so if it is empty it is not clickable?

Reply:
The `TextField` now also opens the file picker when clicked while not having uploaded anything.
Additionally the button text is now colored as a link.